app/controllers/katello/api/v2/alternate_content_sources_controller.rb

```diff
@@ -98,7 +99,7 @@ def refresh
     protected
 
     def acs_params
-      keys = [:name, :label, :base_url, {subpaths: []}, {smart_proxy_ids: []}, :content_type, :alternate_content_source_type,
+      keys = [:name, :label, :base_url, {subpaths: []}, {smart_proxy_ids: []}, {smart_proxy_names: []}, :content_type, :alternate_content_source_type,
```

Comment:
`smart_proxy_names` is making it into the ACS model instantiation up above. We need to ensure it's filtered out like `smart_proxy_ids` are:

```
21:31:30 rails.1   |     59: def create
21:31:30 rails.1   |     60:   find_smart_proxies
21:31:30 rails.1   |     61:   binding.pry
21:31:30 rails.1   |  => 62:   @alternate_content_source = ::Katello::AlternateContentSource.new(acs_params.except(:smart_proxy_ids))
21:31:30 rails.1   |     63:   fail HttpErrors::NotFound, _("Couldn't find smart proxies with id '%s'") % params[:smart_proxy_ids].to_sentence if @smart_proxies.empty?
21:31:30 rails.1   |     64: 
21:31:30 rails.1   |     65:   sync_task(::Actions::Katello::AlternateContentSource::Create, @alternate_content_source, @smart_proxies)
21:31:30 rails.1   |     66:   @alternate_content_source.reload
21:31:30 rails.1   |     67:   respond_for_create(resource: @alternate_content_source)
21:31:30 rails.1   |     68: end
21:31:30 rails.1   | 
21:31:33 rails.1   | [2] pry(#<Katello::Api::V2::AlternateContentSourcesController>)> acs_params.except(:smart_proxy_ids)
21:31:33 rails.1   | 2022-05-12T21:31:33 [D|app|2def983e] Unpermitted parameter: :description
21:31:33 rails.1   | => {"name"=>"asdfasdf", "base_url"=>"https://google.com", "subpaths"=>[""], "smart_proxy_names"=>["centos8-katello-devel.cannolo.example.com"], "content_type"=>"yum", "alternate_content_source_type"=>"custom", "ssl_ca_cert_id"=>"", "verify_ssl"=>false}
21:31:35 rails.1   | [3] pry(#<Katello::Api::V2::AlternateContentSourcesController>)> 2022-05-12T21:31:35 [I|app|e7f5091f] Started GET "/notification_recipients" for 192.168.122.1 at 2022-05-12 21:31:35 +0000
21:31:36 rails.1   | 2022-05-12T21:31:36 [I|app|e7f5091f] Processing by NotificationRecipientsController#index as JSON
21:31:36 rails.1   | 2022-05-12T21:31:36 [D|not|e7f5091f] Cache Hit: notification, reading cache for notification-4
21:31:36 rails.1   | 2022-05-12T21:31:36 [D|app|e7f5091f] Body: {"notifications":[]}
21:31:36 rails.1   | 2022-05-12T21:31:36 [I|app|e7f5091f] Completed 200 OK in 125ms (Views: 1.1ms | ActiveRecord: 6.1ms | Allocations: 1290343)
21:31:36 rails.1   |  e7f5091f | 
21:31:36 rails.1   |  e7f5091f | 
21:31:39 rails.1   | 2022-05-12T21:31:39 [D|app|2def983e] Unpermitted parameter: :description
21:31:39 rails.1   | 2022-05-12T21:31:39 [E|app|2def983e] ActiveRecord::InverseOfAssociationNotFoundError: Could not find the inverse association for smart_proxy (:smart_proxy_alternate_content_sources in SmartProxy)
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/reflection.rb:241:in `check_validity_of_inverse!'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/reflection.rb:474:in `check_validity!'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/association.rb:43:in `initialize'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations.rb:237:in `new'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations.rb:237:in `association'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/builder/association.rb:108:in `smart_proxy='
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:50:in `public_send'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:50:in `_assign_attribute'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:43:in `block in _assign_attributes'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:42:in `each'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:42:in `_assign_attributes'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/attribute_assignment.rb:21:in `_assign_attributes'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:35:in `assign_attributes'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/core.rb:324:in `initialize'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/inheritance.rb:70:in `new'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/inheritance.rb:70:in `new'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/reflection.rb:157:in `build_association'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/association.rb:320:in `build_record'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/collection_association.rb:337:in `build_record'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/collection_association.rb:108:in `build'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/has_many_through_association.rb:64:in `build_through_record'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/has_many_through_association.rb:44:in `block in concat_records'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/has_many_through_association.rb:43:in `each'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/has_many_through_association.rb:43:in `concat_records'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/collection_association.rb:118:in `concat'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/has_many_through_association.rb:21:in `concat'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/collection_association.rb:409:in `replace_records'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/collection_association.rb:249:in `replace'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/collection_association.rb:71:in `ids_writer'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/associations/builder/collection_association.rb:65:in `smart_proxy_ids='
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/app/models/concerns/has_many_common.rb:64:in `block in has_many_names_for'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:50:in `public_send'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:50:in `_assign_attribute'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:43:in `block in _assign_attributes'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:42:in `each'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:42:in `_assign_attributes'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/attribute_assignment.rb:21:in `_assign_attributes'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activemodel-6.0.4.8/lib/active_model/attribute_assignment.rb:35:in `assign_attributes'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/core.rb:324:in `initialize'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/inheritance.rb:70:in `new'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/inheritance.rb:70:in `new'
21:31:39 rails.1   |  2def983e | /home/vagrant/katello/app/controllers/katello/api/v2/alternate_content_sources_controller.rb:62:in `create'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_controller/metal/basic_implicit_render.rb:6:in `send_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/abstract_controller/base.rb:195:in `process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_controller/metal/rendering.rb:30:in `process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/abstract_controller/callbacks.rb:42:in `block in process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/callbacks.rb:112:in `block in run_callbacks'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/app/controllers/concerns/foreman/controller/timezone.rb:10:in `set_timezone'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/callbacks.rb:121:in `block in run_callbacks'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/app/models/concerns/foreman/thread_session.rb:32:in `clear_thread'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/callbacks.rb:121:in `block in run_callbacks'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/app/controllers/concerns/foreman/controller/topbar_sweeper.rb:12:in `set_topbar_sweeper_controller'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/callbacks.rb:121:in `block in run_callbacks'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/audited-4.10.0/lib/audited/sweeper.rb:14:in `around'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/callbacks.rb:121:in `block in run_callbacks'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/audited-4.10.0/lib/audited/sweeper.rb:14:in `around'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/callbacks.rb:121:in `block in run_callbacks'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/callbacks.rb:139:in `run_callbacks'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/abstract_controller/callbacks.rb:41:in `process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_controller/metal/rescue.rb:22:in `process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_controller/metal/instrumentation.rb:33:in `block in process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/notifications.rb:180:in `block in instrument'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/notifications.rb:180:in `instrument'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_controller/metal/instrumentation.rb:32:in `process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_controller/metal/params_wrapper.rb:245:in `process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/railties/controller_runtime.rb:27:in `process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/katello/app/controllers/katello/concerns/api/api_controller.rb:50:in `process_action'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/abstract_controller/base.rb:136:in `process'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionview-6.0.4.8/lib/action_view/rendering.rb:39:in `process'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_controller/metal.rb:190:in `dispatch'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_controller/metal.rb:254:in `dispatch'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/routing/route_set.rb:50:in `dispatch'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/routing/route_set.rb:33:in `serve'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/routing/mapper.rb:18:in `block in <class:Constraints>'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/routing/mapper.rb:48:in `serve'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/journey/router.rb:49:in `block in serve'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/journey/router.rb:32:in `each'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/journey/router.rb:32:in `serve'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/routing/route_set.rb:834:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/railties-6.0.4.8/lib/rails/engine.rb:527:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/railties-6.0.4.8/lib/rails/railtie.rb:190:in `public_send'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/railties-6.0.4.8/lib/rails/railtie.rb:190:in `method_missing'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/routing/mapper.rb:19:in `block in <class:Constraints>'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/routing/mapper.rb:48:in `serve'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/journey/router.rb:49:in `block in serve'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/journey/router.rb:32:in `each'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/journey/router.rb:32:in `serve'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/routing/route_set.rb:834:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/katello/lib/katello/middleware/organization_created_enforcer.rb:18:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/katello/lib/katello/middleware/event_daemon.rb:10:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/static.rb:126:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/static.rb:126:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/apipie-dsl-2.4.0/lib/apipie_dsl/static_dispatcher.rb:67:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/apipie-rails-0.5.20/lib/apipie/static_dispatcher.rb:66:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/bullet-7.0.1/lib/bullet/rack.rb:15:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/apipie-rails-0.5.20/lib/apipie/extractor/recorder.rb:137:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/lib/foreman/middleware/libvirt_connection_cleaner.rb:9:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/lib/foreman/middleware/telemetry.rb:10:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/apipie-rails-0.5.20/lib/apipie/middleware/checksum_in_headers.rb:27:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-openid-1.4.2/lib/rack/openid.rb:98:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/tempfile_reaper.rb:15:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/etag.rb:27:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/conditional_get.rb:40:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/head.rb:12:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/http/content_security_policy.rb:18:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/lib/foreman/middleware/logging_context_session.rb:22:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/session/abstract/id.rb:266:in `context'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/session/abstract/id.rb:260:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/cookies.rb:654:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activerecord-6.0.4.8/lib/active_record/migration.rb:567:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/callbacks.rb:27:in `block in call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/callbacks.rb:101:in `run_callbacks'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/callbacks.rb:26:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/executor.rb:14:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/actionable_exceptions.rb:18:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/debug_exceptions.rb:32:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/show_exceptions.rb:33:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/railties-6.0.4.8/lib/rails/rack/logger.rb:37:in `call_app'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/railties-6.0.4.8/lib/rails/rack/logger.rb:28:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/sprockets-rails-3.4.2/lib/sprockets/rails/quiet_assets.rb:13:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/lib/foreman/middleware/logging_context_request.rb:11:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/remote_ip.rb:81:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/request_id.rb:27:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/katello/lib/katello/prevent_json_parsing.rb:12:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/method_override.rb:24:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/runtime.rb:22:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/activesupport-6.0.4.8/lib/active_support/cache/strategy/local_cache_middleware.rb:29:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/executor.rb:14:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/static.rb:126:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/sendfile.rb:110:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/actionpack-6.0.4.8/lib/action_dispatch/middleware/host_authorization.rb:103:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/secure_headers-6.3.3/lib/secure_headers/middleware.rb:11:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/railties-6.0.4.8/lib/rails/engine.rb:527:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/railties-6.0.4.8/lib/rails/railtie.rb:190:in `public_send'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/railties-6.0.4.8/lib/rails/railtie.rb:190:in `method_missing'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/urlmap.rb:74:in `block in call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/urlmap.rb:58:in `each'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/rack-2.2.3/lib/rack/urlmap.rb:58:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/puma-5.6.4/lib/puma/configuration.rb:252:in `call'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/puma-5.6.4/lib/puma/request.rb:77:in `block in handle_request'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/puma-5.6.4/lib/puma/thread_pool.rb:340:in `with_force_shutdown'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/puma-5.6.4/lib/puma/request.rb:76:in `handle_request'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/puma-5.6.4/lib/puma/server.rb:441:in `process_client'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/puma-5.6.4/lib/puma/thread_pool.rb:147:in `block in spawn_thread'
21:31:39 rails.1   |  2def983e | /home/vagrant/foreman/.vendor/ruby/2.7.0/gems/logging-2.3.0/lib/logging/diagnostic_context.rb:474:in `block in create_with_logging_context'
21:31:39 rails.1   | 2022-05-12T21:31:39 [I|app|2def983e] Completed 500 Internal Server Error in 13061ms (Views: 0.3ms | ActiveRecord: 21.5ms | Allocations: 7542592)
21:31:39 rails.1   |  2def983e | 
```

Reply:
> `smart_proxy_names` is making it into the ACS model instantiation up above. We need to ensure it's filtered out like `smart_proxy_ids` are:

This should be fixed.

>When the creation fails, all of the wizard information needs to be reentered.

Sadly, that is the flow on a wizard. We could add a few validations on the wizard to reduce these situations.

Fields don't have past information being suggested by the browser. For example, during repo creation, the browser suggests upstream URLs that I've entered before. I don't see the same happening here.
-> I suppose this could be a browser issue, but I'm not sure how that stuff works with React wizards.

I have been seeing suggestions in text inputs. Not text area inputs but that is normal.

Reply:
Hm maybe firefox is just being weird for me then!  Testing the push now.